.github/workflows/run_fvt_tests.yaml

```diff
@@ -182,6 +176,64 @@ jobs:
             fi
             exit $RESULT
 
+        - name: Delete deployment
+          run: |
+            helm delete emqx
+
+        - name: Test Deploy with PVC
+          run: |
+            helm install emqx-pvc \
+                --debug \
+                --set image.repository=${TARGET} \
+                --set image.pullPolicy=Never \
+                --set emqxAclConfig="" \
+                --set image.pullPolicy=Never \
+                --set emqxConfig.EMQX_ZONES__DEFAULT__MQTT__RETRY_INTERVAL=2s \
+                --set emqxConfig.EMQX_ZONES__DEFAULT__MQTT__MAX_TOPIC_ALIAS=10 \
+                --set persistence.enabled=true \
+                --set persistence.storageClass=local-path \
```

Comment:
Maybe we could deploy the `storageClass` directly on line 115 and run the tests instead of creating a new deployment

Reply:
Do we need to cover both with/without PVC cases?

Reply:
> Do we need to cover both with/without PVC cases?

I think PVC tests can cover tests that do not contain PVC, so we don’t need without PVC cases